### DISCUSSION TITLE
Can I remove the lightbox from pretix widget?

### DISCUSSION THREAD

**Discussion by davidpt** [pretix/pretix#3959] [Q&A] [answered]
Hi there, I am using pretix widget in a Squarespace website and I'm facing some issues when users try to purchase tickets in the website while on desktop. On desktop when users click buy a lightbox is opened in the same page, which shows an error message saying the server that hosts the ticketing refused to establish connection. However, when in mobile a new tab is opened and in that the user is redirected to the server that host the ticketing system. My question is, can I make the new tab and redirect the default behavior? I'm guessing the hosting server could also be tweaked but for now is there an easy way to change the mentioned behavior of the buy button?

**Comment by raphaelm** [comment, marked as answer]
Have you consulted the documentation yet? :wink: 
https://docs.pretix.eu/en/latest/user/events/widget.html#always-open-a-new-tab

**Reply by davidpt** [reply]
Can't believe I went to that page and missed that.. Thank you Raphael, that solved my issue.
